spk/radarr/src/config.xml

```diff
@@ -1,8 +1,6 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <Config>
   <Branch>master</Branch>
-  <UpdateMechanism>BuiltIn</UpdateMechanism>
   <LaunchBrowser>False</LaunchBrowser>
-  <Port>8310</Port>
```

Comment:
I like to have the port definition here (despite it is the default).
This is good for documentation (and prevents troubles, when, for what reason ever, the default should change).
Only the Port should be kept, not the SslPort (except the package would support the ssl port too).

Reply:
hmm, okay i'll add it in but we need to be consistent with the *arr apps. the release we worked on for sonarr doesn't have the port number